examples/golang/docker-compose-go.yaml

```diff
@@ -6,7 +6,8 @@ services:
       context: ../shared/build
     command: >
       bash -c "
-      bazel build  //examples/golang/simple:simple.so
+      chown envoybuild /output
```

Comment:
you need to do this in the entrypoint - here the user is already lacking permissions

Reply:
Done. got, thanks.